### ISSUE TITLE
Different font sizes for list and app

### ISSUE THREAD

**Issue by mpcjanssen** [mpcjanssen/simpletask-android#260]
Is it possible to have separated font size settings for the list only and for the application in general ?
E.g. Huge font size for the list and Medium font size for the application.

**Comment by smichel17** [comment]
I think the list of settings is already pretty long. If you want to add more, I'd encourage you to organize them into sub-menus. Actually, that might be worth doing regardless.

I think there's a bit of room for improved categories. It's probably not worth doing in the current flat structure, but if you're interested in making sub-menus, I'd be happy to put together a list of updated categories.

**Comment by mpcjanssen** [comment]
@smichel17  Agreed on the length. I would like to reduce the size and add better sectioning. If you have a suggestions that would be appreciated.

**Comment by smichel17** [comment]
Working on it now. Quick question @mpcjanssen: There are two overall directions you could take this: either have widgets inherit all possible settings from the app's internal settings (advantage: fewer settings) or let ALL applicable settings be customized for widgets (advantage: flexibility).

If you choose the former, I would group widget settings with the related app setting (ex: widget theme and app theme together under 'Appearance') -- an intermediary step before removing widget settings in favor of inheriting them from the app.

If you choose the latter, I would group all widget settings together under their own menu -- as an intermediary step towards removing them from the app's own settings and into the widget creation dialogue, so they can be set on a widget-by-widget basis (inheriting the app's settings by default).

Food for thought: If you went the latter route, this could eventually pave the way for breaking the widget into a separate app, like [calendar widget](https://f-droid.org/repository/browse/?fdfilter=calendar+widget&fdid=com.plusonelabs.calendar), which would enable you to fix #282.

That's very long-term, though. The (totally functional) intermediate is just having global widget settings grouped together.

**Comment by smichel17** [comment]
Here's what I'm thinking at the moment. For now, just review the sorting. As per material design guidelines, this includes the ordering of items within a section (by importance) too. I want to do another pass at phrasing and other details, like which settings should be a check box vs a switch, so don't worry about that yet.

@mpcjanssen it might be worth renaming this issue because it's much more comprehensive than a single setting now.

Legend:
- Ending with a colon indicates it's a section: (as opposed to a top-level menu item).
- Bold is just drawing attention to **changes in phrasing.**
- Italics indicate _secondary text_, which is only recorded if there are changes to be made, so I won't bother bolding them too.
- Brackets indicate that this should show [current status].
- Strikethrough ~~is obvious~~ means what you think it does
- Parentheses are notes for clarification (for you)

---

Widget:
- ~~Widget~~ theme  
_[Dark]_
- ~~Widget~~ header transparency
- ~~Widget~~ background transparency
- ~~Datebar in widget~~ Show datebar  
_Show dates on a separate line._

Appearance: (purely cosmetic)
- Application theme  
_[Light]_
- ~~Change~~ **F**ont size  
_[Medium]_
- Color due dates
- Show hints
- Use todo.txt terms

Information Display: (affects what information you get from the app, but not how you interact with it)
- Task text ellipsizing
- Show todo path
- Show txt files only
- ~~Datebar in main view~~ Show datebar

User Interface: (affects how you interact with the app, but not how the app behaves)
- Show completion checkbox
- Show confirmation dialogs
- Use calendar for date entry
- Fast scrolling
- Open landscape drawers

Behavior: (Possible rename: "Functionality")
- Task creation:
  - Date new tasks  
  - Capitalize new tasks
  - Append new tasks
- Auto archive
- Back button
- Prio in completed tasks
- Edit shared task**s**  
_Edit tasks shared to simpletask before saving them._
- Background tag
- File link root
- Windows line breaks

Experimental:
- Calendar reminders
- Back clears filter
- View log

Donate

---

The following settings are (re)moved. However, if you want to split this into two steps (reorganize and then remove), they can be combined into an "other" category as below. Put this category after Experimental / before Donate.

Other:
- ~~Simpletask cloudless v7.0.14 (3024)~~ => Help screen as per [material guidelines](https://www.google.com/design/spec/patterns/settings.html#settings-usage)
- ~~Todo history~~ => Main context menu.
- ~~Sort case-sensitive~~ => Filter settings
- ~~Create is threshold~~ => Filter settings
- ~~Archive now~~ => removed. (A part of the app's main workflow does not belong in settings.)
